docs/reference/search/multi-search.asciidoc

```diff
@@ -90,26 +96,58 @@ also provides support for templates. Submit them like follows:
 
 [source,js]
 -----------------------------------------------
-$ cat requests
-{"index" : "main"}
+GET _msearch/template
+{"index" : "twitter"}
+{ "inline" : "{ \"query\": { \"match\": { \"message\" : \"{{keywords}}\" } } } }", "params": { "query_type": "match", "keywords": "some message" } }
+{"index" : "twitter"}
 { "inline" : "{ \"query\": { \"match_{{template}}\": {} } }", "params": { "template": "all" } }
-{"index" : "main"}
-{ "inline" : "{ \"query\": { \"match_{{template}}\": {} } }", "params": { "template": "all" } }
-
-$ curl -XGET localhost:9200/_msearch/template --data-binary @requests; echo
 -----------------------------------------------
+// CONSOLE
+// TEST[setup:twitter]
 
-for inline templates. Alternatively for stored templates:
+for inline templates.
+
+You can also create search templates:
+
+[source,js]
+------------------------------------------
+POST /_search/template/my_template_1
+{
+    "template": {
+        "query": {
+            "match": {
+                "message": "{{query_string}}"
+            }
+        }
+    }
+}
+------------------------------------------
+// CONSOLE
```

Comment:
If you setup twitter here and then `// TEST[continued]` on the next two snippets then you'll get a "more real" test, I think.